WORKSPACE

```diff
@@ -40,29 +63,32 @@ go_repository(
     name = "io_k8s_apimachinery",
     build_file_generation = "on",
     build_file_name = "BUILD.bazel",
-    commit = "b166f81f5c4c88402ae23a0d0944c6ad08bffd3b",
+    build_file_proto_mode = "disable",
+    commit = "18a564baac720819100827c16fdebcadb05b2d0d",
```

Comment:
I guess these SHAs (this and the subsequent ones) were all just matched to the ones in k/k? Can that be a single comment above this rule for all the dependencies that explains how the SHAs were chosen ?

Reply:
These were based on the result of running `dep` and seeing which.

k/ doesn't have a good solution for vendored dependency version management at this point. I don't want to solve that problem right now for this repository, and I think it's important to get vendoring in soon in order to unblock development.

Reply:
I wasn't expecting you to solve the overall k8s vendoring problem and I understand getting this in is critical to unblock development. I was just expecting a comment in this file (or probably in a contributor doc or somewhere) explaining how these SHAs were generated and how could one update them in the future: links to files to copy from, commands to run, etc. Because before this comment, the PR looked like a black box. The top-level comment helps, if that is all one has to do to update all these SHAs.

Reply:
OK, that makes sense. I've added a brief comment here as well to explain how these were selected, but since this is quickly going to be replaced by #58, I didn't flesh out the comment in full detail.